egov-indexer/open-hearing-indexer.yml

```
@@ -0,0 +1,33 @@
+  mappings: 
+    - topic: open-hearing-topic
+      configKey: INDEX
+      indexes: 
+        - name: open-hearing-index
+          type: general
+          id: $.hearingUuid
+          isBulk: false
+          jsonPath: $
+          customJsonMapping: 
+            indexMapping: {"Data":{"hearingDetails":{}}}
+            fieldMapping: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)